src/main/java/picard/util/LiftoverUtils.java

```diff
@@ -192,6 +207,81 @@ protected static GenotypesContext fixGenotypes(final GenotypesContext originals,
     }
 
     /**
+     * method to swap the reference and alt alleles of a bi-allelic, SNP
+     *
+     * @param vc
+     * @return
+     */
+    public static VariantContext swapRefAlt(final VariantContext vc, final Collection<String> annotationsToReverse, final Collection<String> annotationsToDrop) {
+        //TODO: assert vc is a biallelic snp
+
+        final VariantContextBuilder swappedBuilder = new VariantContextBuilder(vc);
+
+        swappedBuilder.attribute(SWAPPED_ALLELES, true);
+
+        // You need to get the baseString in order to forget who's reference and who's variant
+        swappedBuilder.alleles(Arrays.asList(vc.getAlleles().get(1).getBaseString(), vc.getAlleles().get(0).getBaseString()));
+
+
+        final Map<Allele, Allele> alleleMap = new HashMap<>();
+        for (int idx = 0; idx < vc.getAlleles().size(); idx++) {
+            // The index of the allele changed from 1 to 0 and vice versa, but the bases remain the same.
+            alleleMap.put(vc.getAlleles().get(idx), swappedBuilder.getAlleles().get(1 - idx));
+        }
+
+        final GenotypesContext swappedGenotypes = GenotypesContext.create(vc.getGenotypes().size());
+        for (final Genotype genotype : vc.getGenotypes()) {
+            final List<Allele> swappedAlleles = new ArrayList<>();
+            for (final Allele allele : genotype.getAlleles()) {
+                if (allele.isNoCall()) {
+                    swappedAlleles.add(allele);
+                } else {
+                    swappedAlleles.add(alleleMap.get(allele));
+                }
+            }
+            // Flip AD
+            final GenotypeBuilder builder = new GenotypeBuilder(genotype).alleles(swappedAlleles);
+            if (genotype.hasAD() && genotype.getAD().length == 2) {
+                final int[] origAD = genotype.getAD();
+                builder.AD(new int[]{origAD[1], origAD[0]});
+            }
+
+            //Flip PL
+            if (genotype.hasPL() && genotype.getPL().length == 3) {
+                final int[] origPL = genotype.getPL();
+                builder.PL(new int[]{origPL[2], origPL[1], origPL[0]});
+            }
+            swappedGenotypes.add(builder.make());
+        }
+        swappedBuilder.genotypes(swappedGenotypes);
+
+        for (String key : vc.getAttributes().keySet()) {
+            if (annotationsToDrop.contains(key)) {
+                swappedBuilder.rmAttribute(key);
+            }
+
+            if (annotationsToReverse.contains(key)) {
+                final double afLikeAttribute = vc.getAttributeAsDouble(key, -1);
+
+                if (afLikeAttribute == -1) {
+                    log.warn("Trying to reverse attribute " + key +
+                            "but found value that parsable as a double (" + vc.getAttribute(key, "") +
+                            ") in variant " + vc + "results might be wrong.");
```

Comment:
Missing a space before `results`, also looks like a typo, extra `might be wrong.` here and below.